build/terraform/performance/module.tf

```diff
@@ -53,8 +53,8 @@ module "gke_standard_cluster" {
   location = each.value[0]
   overrideName = format("standard-performance-test-cluster-%s", replace(each.key, ".", "-"))
   releaseChannel = each.value[1]
-  machineType = "n2-standard-2"
-  initialNodeCount = 300
+  machineType = "e2-standard-4"
```

Comment:
This is going to double the number of cores in the test cluster. Do you have autoscaling enabled to scale the number of nodes down when it isn't in use?

Reply:
Sorry I didn't make it quite clear in the PR description. The current test cluster is already using type "e2-standard-4" actually. The code here drifted from the fact by some mistake. So the only actual change in this PR is to increase the size to 310. 
Good point to enable autoscaling. I will test and make a quick follow up PR for it.